test/passing/js_source.ml.ref

```diff
@@ -3649,20 +3649,25 @@ end
 let ssmap =
   (module SSMap
   : MapT
-    with type key = string and type data = string and type map = SSMap.map)
+    with type key = string
+     and type data = string
+     and type map = SSMap.map)
 ;;
 
 let ssmap =
   (module struct
     include SSMap
   end
   : MapT
-    with type key = string and type data = string and type map = SSMap.map)
+    with type key = string
+     and type data = string
+     and type map = SSMap.map)
 ;;
 
 let ssmap :
-    (module MapT with type key = string and type data = string and type map = SSMap.map)
-    =
+    (module MapT with type key = string
+                  and type data = string
+                  and type map = SSMap.map) =
```

Comment:
So far no success breaking before `with` without introducing more regressions elsewhere.

Reply:
What kind of regressions ?

Reply:
You can look at the diff of my last commit 2d68308, there are net improvements (even if we break more lines sometimes) but for some cases I'm not sure if it's really better